ts/src/profilers/profile-serializer.ts

```diff
@@ -170,7 +170,9 @@ function createSampleCountValueType(table: StringTable):
  */
 function createTimeValueType(table: StringTable): perftools.profiles.ValueType {
   return new perftools.profiles.ValueType({
-    type: table.getIndexOrAdd('time'),
+    // TODO: change type back to time.
```

Comment:
Java agent puts the time metric in the wall profiles named as "wall", so I'd remove the TODO and leave the metric name as "wall" here after all.

BTW, no test changes needed?

Reply:
Forgot to run the tests -- the tests have now been updated.
Removing TODO now.

Reply:
Done.